packages/root-cms/ui/components/DocEditor/DocEditor.tsx

```diff
@@ -840,7 +860,30 @@ DocEditor.ArrayField = (props: FieldProps) => {
             key={key}
             open={newlyAdded.includes(key) || itemInDeeplink(key)}
           >
-            <summary className="DocEditor__ArrayField__item__header">
+            <summary
+              className="DocEditor__ArrayField__item__header"
+              tabIndex={0}
+              id={`summary-for-${props.deepKey}.${order[i]}`}
+              onKeyDown={(e: KeyboardEvent) => {
```

Comment:
since there's a lot of logic here, let's move this callback into a function so that there's not so much inlining for this prop value.